odoo/addons/base/res/res_partner_view.xml

```diff
@@ -519,9 +519,10 @@
                                         <li t-if="record.parent_id.raw_value and !record.function.raw_value"><field name="parent_id"/></li>
                                         <li t-if="!record.parent_id.raw_value and record.function.raw_value"><field name="function"/></li>
                                         <li t-if="record.parent_id.raw_value and record.function.raw_value"><field name="function"/> at <field name="parent_id"/></li>
-                                        <li t-if="record.city.raw_value and !record.country_id.raw_value"><field name="city"/></li>
-                                        <li t-if="!record.city.raw_value and record.country_id.raw_value"><field name="country_id"/></li>
-                                        <li t-if="record.city.raw_value and record.country_id.raw_value"><field name="city"/>, <field name="country_id"/></li>
+                                        <li t-if="record.city.raw_value or record.country_id.raw_value">
+                                            <t t-if="record.city.raw_value"><field name="city"/><t t-if="record.country_id.raw_value">, </t></t>
```

Comment:
But that means that if there is a country set but no city, the result will be: `, Belgium`. Right?

Wouldn't this be ok:
```suggestion
                                            <t t-if="record.city.raw_value"><field name="city"/><t t-if="record.city.raw_value && record.country_id.raw_value">, </t></t>
```

Reply:
@nim-odoo Its the same, no? I wanted to make it shorter so I put the country condition for the comma inside the city <t> condition.

Reply:
indeed, I missed that